test/test-cid.js

```diff
@@ -243,12 +251,19 @@ describe('CID', () => {
     }
     invalid.forEach(mapper)
 
-    const invalidVersions = [-1, 2]
+    const invalidVersions = [-1, 2, 'one', false, null, {}, []]
     mapper = i => {
       const name = `new CID(${i}, 112, buffer)`
-      test(name, () => testThrowAny(new CID(i, 112, hash)))
+      test(name, () => testThrowAny(() => new CID(i, 112, hash)))
```

Comment:
wasn't being tested properly, I guess this is where the error message checking of `throws()` comes in handy